internal/pkg/v2/infrastructure/redis/queries.go

```diff
@@ -30,6 +30,17 @@ func getObjectById(conn redis.Conn, id string, out interface{}) errors.EdgeX {
 	return nil
 }
 
+func getObjectByHash(conn redis.Conn, hash string, field string, out interface{}) errors.EdgeX {
```

Comment:
add comment for the function in util package

Reply:
Comment added.